xtremio.py

```diff
@@ -925,6 +925,19 @@ def _get_iscsi_properties(self, lunmap):
                       'target_luns': [lunmap['lun']] * len(portals)}
         return properties
 
+    def terminate_connection(self, volume, connector, **kwargs):
+        (super(XtremIOISCSIDriver, self)
+         .terminate_connection(volume, connector, **kwargs))
+        num_vols = (self.client
+                    .num_of_mapped_volumes(self._get_ig_name(connector)))
+        if num_vols > 0:
+            return
+        else:
```

Comment:
I agree, but if we want to delete the initiator group, there is no other way but send the request to XMS.
It is not an error issue but how XtremIO driver should work.
By the way other storage vendor working like this, this is why I think that we should also work as so. Users maybe use multiple backend storage and keep all the storage have the same work will be friendly to user.